CMakeLists.txt

```
@@ -160,6 +164,23 @@ set(UTILS_SRC
   "${HASH_CONTAINERS_DIR}/hash8019_stack.c"
 )
 
+if(${PCURL_SBOX_UNWIND_8})
+  add_definitions("-DPCURL_SBOX_UNWIND_8")
```

`add_definitions` is superseded in a newer version and these flags are private flags, in this case should use `target_compile_definitions`.
```suggestion
  target_compile_definitions(common PRIVATE "PCURL_SBOX_UNWIND_8")
```